RealJenkinsRuleInit/pom.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.jenkins-ci.plugins</groupId>
+        <artifactId>plugin</artifactId>
+        <version>4.17</version>
+        <relativePath />
+    </parent>
+    <groupId>io.jenkins.plugins</groupId>
+    <artifactId>RealJenkinsRuleInit</artifactId>
+    <version>0-SNAPSHOT</version>
+    <packaging>hpi</packaging>
+    <properties>
+        <jenkins.version>2.222.1</jenkins.version>
+        <java.level>8</java.level>
+    </properties>
+    <name>RealJenkinsRule initialization wrapper</name>
+    <build>
+        <plugins>
+            <plugin> <!-- https://maven.apache.org/plugins/maven-dependency-plugin/examples/copying-artifacts.html -->
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <version>3.1.2</version>
+                <executions>
+                    <execution>
+                        <id>copy-installed</id>
+                        <phase>install</phase>
+                        <goals>
+                            <goal>copy</goal>
+                        </goals>
+                        <configuration>
+                            <artifactItems>
+                                <artifactItem>
+                                    <groupId>${project.groupId}</groupId>
+                                    <artifactId>${project.artifactId}</artifactId>
+                                    <version>${project.version}</version>
+                                    <type>${project.packaging}</type>
+                                    <outputDirectory>${project.basedir}/../src/main/resources/org/jvnet/hudson/test</outputDirectory>
+                                    <destFileName>RealJenkinsRuleInit.jpi</destFileName>
```

Comment:
For now, building this artifact manually, since it looks awkward to include it in the main module’s build process (I do not particular want to switch to a reactor build).